frontend/src/main/scala/quasar/data.scala

```diff
@@ -98,6 +98,9 @@ object Data {
   val _obj =
     Prism.partial[Data, ListMap[String, Data]] { case Data.Obj(m) => m } (Data.Obj(_))
 
+  def singletonObj(k: String, v: Data): Data =
```

Comment:
Maybe this doesn't matter if `Data` is going to disappear sooner rather than later, but what if instead of this, we added a companion object to `Obj` which accepted a vararg. Seem like it would make building data objects way more concise in a lot of places in the code.